src/NHibernate/AdoNet/AbstractBatcher.cs

```diff
@@ -135,6 +136,45 @@ protected void Prepare(DbCommand cmd)
 			}
 		}
 
+#if NET6_0_OR_GREATER
+		/// <summary>
+		/// Prepares the <see cref="DbBatch"/> for execution in the database.
+		/// </summary>
+		/// <remarks>
+		/// This takes care of hooking the <see cref="DbBatch"/> up to an <see cref="DbConnection"/>
+		/// and <see cref="DbTransaction"/> if one exists.  It will call <c>Prepare</c> if the Driver
+		/// supports preparing batches.
+		/// </remarks>
+		protected void Prepare(DbBatch batch)
```

Comment:
It feels that this method does not belong to `AbstractBatcher` and should instead be on the `DbBatchBatcher`. This would allow to:
1. Contain logic related to `DbBatch` in `DbBatchBatcher`
2. Have more flexibility in terms of methods